app/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe 'Users controller', type: :request do
+  it 'should create a users page and show a list of users' do
+    get '/users'
+    expect(response).to have_http_status(200)
+    expect(response).to render_template(:index)
+    expect(response.body).to include('This is a list of users')
+  end
+
+  it 'should create a user page and show details about a user' do
+    get '/users/123'
+    expect(response).to have_http_status(200)
+    expect(response).to render_template(:show)
+    expect(response.body).to include('This is a detail about some user')
+  end
+end
```

Comment:
- [ ] Based on the requirement, you should have a `users_spec.rb` and `posts_spec.rb` in `spec/requests/` directory. Kindly remove these files from the `app/controllers/` file path and create a `request` directory in the `spec` directory in the root of your project. Essentially you will have this `spec/requests/` directory structure. 

![image](https://user-images.githubusercontent.com/84156714/218083856-e91bd0a7-7d73-4f20-a4ea-7d4c4b6446ce.png)